### DISCUSSION TITLE
Speaker diarization not working

### DISCUSSION THREAD

**Discussion by naman-statisfy** [deepgram/community#476] [General help] [answered]
I am trying to transcribe an audio file using with 3 different speakers in it. I am using python sdk as mentioned in the website tutorial. I am passing 'diarize' as True. However I am getting 'speaker' field as 0 for all the speakers. Is there anything else that I am missing?
I have attached the audio file if it helps. Thanks
[firstMinute.zip](https://github.com/deepgram/community/files/13542170/firstMinute.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @naman-statisfy, I'm getting similar results on the audio you shared. I see it's lower quality audio, frequency of 5.5 KHz and bit rate of 89 Kbps, which means that the diarizer does not have as much audio information to use in its predictions. Are you able to get a higher quality version of the original audio, or test with a different file? I don't believe you're doing anything incorrectly in this case, the audio itself is just not well suited for diarization.
